### DISCUSSION TITLE
Dark dimmed theme missing from /join/get-started page

### DISCUSSION THREAD

**Discussion by 06000208** [community/community#37005] [Accessibility] [answered]
https://github.com/join/get-started

![firefox_6Wxdyr4vAc](https://user-images.githubusercontent.com/52764066/197373756-ad49bd3f-631c-4136-a044-b603cca30e61.png)

You are redirected to this page if you add additional email addresses to your account, hence me finding this.

Dunno if this was the right place to report but figured I'd try and find someplace to post it.

**Comment by DSDmark** [comment, marked as answer]
Hey @06000208,

If you want to report bug then visit there:-
[**Bug-Report**](https://support.github.com/contact/bug-report)

---

Or if you want some things else then visit there:-
[**Github-support**](https://support.github.com/)

**Reply by 06000208** [reply]
Thanks for the proper link, I'll submit a bug report.
